### PR TITLE
Add initial Service provider section for DataHub

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary 

Adding initial doc for service providers to install oneprovider to join EGI DataHub

### CODE REVIEW

File: content/providers/datahub/oneprovider/_index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: "DataHub OneProvider"
+linkTitle: "OneProvider"
+type: docs
+description: "Documentation for installation/configuration of OneProvider to join EGI DataHub spaces"
+weight: 20
+---
+
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:8]:
```suggestion
```

File: content/providers/datahub/oneprovider/_index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Oneprovider
+  - RAM: 32GB
+  - CPU: 8 vCPU
+  - Disk: 50GB SSD
+  - To be adjusted for the dataset and usage scenario
+- For high IOPS
+  - High performance backend storage (CEPH)
+  - Low latency network
+
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:25]:
```suggestion
```

File: content/providers/datahub/oneprovider/_index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+## Installation and attach empty storage to the EGI DataHub
+
+The installation of a new OneProvider is performed using the an installation script used `onedatify` which
+will setup the components using Docker and Docker-compose
+
+The simple installation script can be generated from the EGI DataHub web interface.
+
+First you need to login to the EGI DataHub and from the Data menu you either select  an existing space
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:34]:
```suggestion
First you need to login to the EGI DataHub and from the Data menu you either select an existing space 
```

File: content/providers/datahub/oneprovider/_index.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+First you need to login to the EGI DataHub and from the Data menu you either select  an existing space 
+or create a new one.
+
+You can then select on the space menu the Providers section and click on the Add Support button on the top
+right corner 
+
+![image](add-support-oneprovider.png)
+
+
+You should then select on the page the tab : `Deploy your own provider` and there you will have just to copy
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:43]:
```suggestion
You should then select on the page the tab: `Deploy your own provider` and there you will have just to copy 
```

File: content/providers/datahub/oneprovider/_index.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+or create a new one.
+
+You can then select on the space menu the Providers section and click on the Add Support button on the top
+right corner 
+
+![image](add-support-oneprovider.png)
+
+
+You should then select on the page the tab : `Deploy your own provider` and there you will have just to copy 
+the command having already the correct parameters for the OneZone to use ( Datahub.egi.eu ) and the
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:44]:
```suggestion
the command having already the correct parameters for the OneZone to use (datahub.egi.eu) and the 
```

File: content/providers/datahub/oneprovider/_index.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+![image](add-support-oneprovider.png)
+
+
+You should then select on the page the tab : `Deploy your own provider` and there you will have just to copy 
+the command having already the correct parameters for the OneZone to use ( Datahub.egi.eu ) and the 
+space to join.
+
+![image](onedatify-oneprovider.png)
+
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:48]:
```suggestion
```

File: content/providers/datahub/oneprovider/_index.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+The progress can be monitored on a separate terminal using the following command:
+
+`onedatify logs`
+
+After the deployment is complete, the following message will be shown, with connection details for administration panel for the Oneprovider instance:
+
+![image](onedatify_step_5.png)
+
+The WebUI Admin portal at port 9443 can be used to administer the OneProvider installation
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:68]:
```suggestion
The WebUI Admin portal at port 9443 can be used to administer the OneProvider installation.
```

File: content/providers/datahub/oneprovider/_index.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+##  Installation and expose existing data to the EGI DataHub
+
+The installation of a new OneProvider to expose existing data sets to an EGI DataHub space is similar to the installation with an 
+empty storage.
+
+When adding support to an existing or new space you should select from the EGi DataHub GUI  the tab : `Expose Existing dat set` and there you will have just to copy  the command having already the correct parameters for the OneZone to use ( Datahub.egi.eu ) and the space to join.
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:75]:
```suggestion
When adding support to an existing or new space you should select from the EGI DataHub GUI the tab : `Expose Existing dat set` and there you will have just to copy the command having already the correct parameters for the OneZone to use (datahub.egi.eu) and the space to join.
```

File: content/providers/datahub/oneprovider/_index.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+![image](add-support-oneprovider-expose.png)
+
+### Run the command on the host
+
+Paste the copied command in the terminal on the OneProvider machine (as superuser) and follow the instructions as for the case of an empty storage.
+
+The only difference is that at the end of the installation and configuration process the existing files will be automatically imported to the OneProvider. You can monitor the import activity from the WebUI Admin panel at port 9443
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:83]:
```suggestion
The only difference is that at the end of the installation and configuration process the existing files will be automatically imported to the OneProvider. You can monitor the import activity from the WebUI Admin panel at port 9443:
```

File: content/providers/datahub/oneprovider/_index.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+![image](onedatify_step_6.png)
+
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:86]:
```suggestion
```

File: content/providers/datahub/oneprovider/_index.md (hunk 8 above)
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:75]:
```suggestion
When adding support to an existing or new space you should select from the EGI DataHub GUI  the tab: `Expose Existing dat set` and there you will have just to copy  the command having already the correct parameters for the OneZone to use (datahub.egi.eu) and the space to join.
```

**Comment by andrea-manzi** [comment]
thanks! I applied your changes

**Pull request merged**